src/components/Footer/index.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Header } from "../";
+import {
+  Container,
+  Row,
+  Link,
+  Logo,
+  Center,
+  Col,
+  Subheading,
+  Heading,
+  TextLink,
+  Icon,
+  TextHeight,
+} from "./styles";
+
+import { ReactComponent as Meta } from "../../assets/media/meta.svg";
+import { ReactComponent as Discord } from "../../assets/media/discord.svg";
+import { ReactComponent as Tiktok } from "../../assets/media/tiktok.svg";
+import { ReactComponent as Instagram } from "../../assets/media/instagram.svg";
+import CKI_Logo from "../../assets/images/cki_logo.png";
+
+function HyperLink(props) {
+  const Icon = props.icon;
+  const link = props.href;
+  return (
+    <li>
+      <Link href={link} target="_blank">
+        <Icon />
+        <div>{props.children}</div>
+      </Link>
+    </li>
+  );
+}
+
+export function Footer() {
+  return (
+    <div className="main-footer">
+      <Container>
+        <Row>
+          {/*column1 */}
+          <ul className="list-unstyled">
+            <Logo src={CKI_Logo} alt="Circle K International Logo" />
+            <Heading>Circle K</Heading>
+            <Subheading>Cal Poly Pomona</Subheading>
+          </ul>
+        </Row>
+        {/* Column2 */}
+        <Row>
+          <Col>
+            <Subheading>
+              <li> Contact Us </li>
+            </Subheading>
+            <p>
+              <Instagram fill="#B1953A" />
+              <TextLink href="https://Instagram.com" target="_blank">
+                cppcki
+              </TextLink>
+            </p>
+            <p>
+              <Meta fill="#B1953A" />
+              <TextLink href="https://Facebook.com" target="_blank">
+                cppcki
+              </TextLink>
+            </p>
+            <p>
+              <Tiktok fill="#B1953A" />
+              <TextLink href="https://Tiktok.com" target="_blank">
+                cppcki
+              </TextLink>
+            </p>
+            <p>
+              <Discord fill="#B1953A" />
+              <TextLink href="https://Discord.com" target="_blank">
+                cppcki
+              </TextLink>
+            </p>
+          </Col>
+        </Row>
+        {/* Column3 */}
+        <Row>
+          <Col>
+            <Subheading>
+              <TextHeight>
+                <p>About Us</p>
+                <p>Events</p>
+                <p>Resources</p>
+                <p>Join</p>
```

Comment:
These should be hypertext links and should be direct to the respective page.

Since we're using a library to handle routes, refer to using `<Link/>` which is imported from `react-router-dom`. If you need an example of how that works refer to the Anchor component.

Also keep the `font-size` as the default.

```js
import { Link } from "react-router-dom";

<Link to="path"/>
```

Reply:
The links are made but is there any specific paths to lead some of them? "About us" leads to the same page, "Events" has nothing, "Join" and "Resources" have pages to link to.

Reply:
Inside of `src/index.js`, there are routes that correspond to the path's name.
```js
        <Routes>
          <Route path="/" element={<Home/>}/>
          <Route path="/about" element={<About/>}/>
          <Route path="/resources" element={<Resources/>}/>
          <Route path="/join" element={<Join/>}/>
        </Routes>
```

So if you want the hyperlink to direct to `about` you would put

```js
<Link to="/about">about us </Link>
```

Here's more docs about how `Link` component works.
https://v5.reactrouter.com/web/api/Link